python/lsst/ts/Dome/dome_csc.py

```diff
@@ -465,86 +481,73 @@ async def statusAMCS(self):
         """LCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the LCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.LCS, self.tel_domeLouvers_status)
+        await self.request_and_send_llc_status(LlcName.LCS, self.tel_louvers)
 
     async def statusLWSCS(self):
         """LWSCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the LWSCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.LWSCS, self.tel_domeLWS_status)
+        await self.request_and_send_llc_status(LlcName.LWSCS, self.tel_lightWindScreen)
 
     async def statusMonCS(self):
         """MonCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the MonCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.MONCS, self.tel_domeMONCS_status)
+        await self.request_and_send_llc_status(LlcName.MONCS, self.tel_interlocks)
 
     async def statusThCS(self):
         """ThCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the ThCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.THCS, self.tel_domeTHCS_status)
+        await self.request_and_send_llc_status(LlcName.THCS, self.tel_thermal)
+
+    async def request_and_send_llc_status(self, llc_name, topic):
+        """Generic method for retrieving the status of a lower level component and publish that on the
+        corresponding telemetry topic.
+
+        Parameters
+        ----------
+        llc_name: `LlcName`
+            The name of the lower level component.
+        topic: SAL topic
+            The SAL topic to publish the telemetry to.
 
-    async def request_and_send_llc_status(self, llc_name, telemetry_function):
+        """
         command = f"status{llc_name.value}"
         status = await self.write_then_read_reply(command=command, parameters={})
         # Store the status for unit tests.
-        self.lower_level_status[llc_name.value] = status[llc_name.value]
-        self.log.info(status)
-        self.convert_telemetry_radians_to_degrees_and_send(
-            status[llc_name.value], telemetry_function, llc_name
-        )
+        self.lower_level_status[llc_name.value] = status[llc_name.value][0]
 
-    def convert_telemetry_radians_to_degrees_and_send(
-        self, telemetry_in_radians, telemetry_function, llc_name
-    ):
         telemetry_in_degrees = {}
+        telemetry_in_radians = status[llc_name.value][0]
         for key in telemetry_in_radians.keys():
-            if key in _KEYS_IN_RADIANS and (
-                llc_name == LlcName.AMCS or llc_name == LlcName.APSCS
-            ):
+            # LWSCS is handled by convert_telemetry_list_radians_to_degrees_and_send
```

Comment:
I think you got rid of `convert_telemetry_list_radians_to_degrees_and_send`? If so, please update or delete this comment. I confess I'm not sure what it is telling me.

Reply:
Thanks for catching that! I removed the comment.